clusters/build-clusters/01_cluster/sshd-bastion/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
```

Comment:
Are those file with the same contents as before?
Moving is fine, no need for copying.
It wont be deleted on `api.ci` after merging even in the case of moving the files over.
We have to manually delete the resources on `api.ci` when the jobs work fine on `build01`.